package/kedro_viz/api/apps.py

```diff
@@ -61,7 +61,9 @@ def create_api_app_from_project(
     app = _create_base_api_app()
     app.include_router(rest_router)
     app.include_router(graphql_router)
-    app.mount("/static", StaticFiles(directory=_HTML_DIR / "static"), name="static")
+
+    if Path(_HTML_DIR).is_dir():
```

Comment:
Maybe add a comment above this line to describe why this check is necessary? I don't think it's immediately clear.

Reply:
done !